cmd/kops/get_cluster.go

```diff
@@ -46,6 +46,21 @@ var (
 	kops get cluster k8s-cluster.example.com`))
 
 	get_cluster_short = i18n.T(`Get one or many clusters.`)
+
+	// Warning for --full.  Since we are not using the template from kubectl
+	// we have to have zero white space before the comment characters otherwise
+	// output to stdout is going to be off.
+	get_cluster_full_warning = i18n.T(`
+//
+//   WARNING: Do not use a '--full' cluster specification to define a Kubernetes installation.
```

Comment:
It is NOT a yaml comment or a JSON comment - deliberately . You want to output text without //?